internal/manifests/config/configmap.go

```diff
@@ -0,0 +1,214 @@
+package config
+
+import (
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/grafana/dskit/kv"
+	"github.com/grafana/dskit/kv/memberlist"
+	"github.com/grafana/dskit/ring"
+	tempoapp "github.com/grafana/tempo/cmd/tempo/app"
+	tempodistributor "github.com/grafana/tempo/modules/distributor"
+	"github.com/grafana/tempo/modules/ingester"
+	"github.com/grafana/tempo/modules/storage"
+	"github.com/grafana/tempo/tempodb"
+	"github.com/grafana/tempo/tempodb/backend"
+	"github.com/grafana/tempo/tempodb/backend/local"
+	"github.com/grafana/tempo/tempodb/encoding/common"
+	"github.com/grafana/tempo/tempodb/wal"
+	"github.com/weaveworks/common/logging"
+	"github.com/weaveworks/common/server"
+	"gopkg.in/yaml.v3"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/os-observability/tempo-operator/api/v1alpha1"
+	"github.com/os-observability/tempo-operator/internal/manifests/manifestutils"
+)
+
+const (
+	kvStore = "memberlist"
+)
+
+// nolint
+func config(tempo v1alpha1.Microservices) (string, error) {
+	logLevel := logging.Level{}
+	logLevel.Set("warn")
+	logFormat := logging.Format{}
+	logFormat.Set("logfmt")
+	cfg := tempoapp.Config{
```

Comment:
Will the `tempoApp.Config` be backward compatible in the long run? Otherwise it could lead to problems when older component images are used.

Reply:
Good comment. I don't know, probably the new config options are ignored when parsing the config. There will be a separate follow up task to improve config handling.

Reply:
new options are not an issue. I thought more about removed and renamed options ^^

Reply:
Also we want to probably support an operator version per tempo version.